tigerpath/majors_and_certificates/majors/NEU_2018.json

```diff
@@ -0,0 +1,348 @@
+{
+    "type": "Major",
+    "name": "Neuroscience",
+    "code": "NEU",
+    "degree": "AB",
+    "year": 2018,
+    "description": "The Princeton Neuroscience Institute offers the neuroscience concentration for undergraduates with a strong interest in pursuing an in-depth study of the brain. Neuroscience is a broad interdisciplinary field requiring rigorous preparation in basic science. Students in this discipline are expected to understand the basic principles and approaches of modern neuroscience. The concentration provides an opportunity for the serious study of molecular, cellular, developmental, and systems neuroscience as it interfaces with cognitive and behavioral research. Since modern neuroscience is relying increasingly on quantitatively sophisticated methods and theory, students are also expected to gain competency in, physics, mathematics and computation. By offering a combination of courses and interdisciplinary research, students who complete the neuroscience concentration will be highly qualified to pursue graduate work at the best neuroscience, psychology or biology graduate programs and will also have completed, in large part, the background requirements to enter medical or veterinary school. Other possible career paths for those with a neuroscience degree can be found here.",
+    "urls": [
+        "https://ua.princeton.edu/academic-units/neuroscience-ab-through-princeton-neuroscience-institute",
+        "http://pni.princeton.edu/"
+    ],
+    "contacts": [
+        {
+            "type": "Departmental Representative",
+            "name": "Asif A. Ghazanfar",
+            "email": "asifg@princeton.edu"
+        },
+        {
+            "type": "Director",
+            "name": "Jonathan D. Cohen",
+            "email": "jdc@princeton.edu"
+        },
+        {
+            "type": "Director",
+            "name": "David W. Tank",
+            "email": "dwtank@princeton.edu"
+        }
+    ],
+    "req_list": [
+        {
+            "name": "Prerequisites",
+            "max_counted": 1,
+            "min_needed": "ALL",
+            "explanation": "To enter the neuroscience concentration, students must have completed NEU 201 Fundamentals of Neuroscience, NEU 202 Introduction to Cognitive Neuroscience, and MAT 103 Calculus I.",
+            "req_list": [
+                {
+                    "name": null,
+                    "max_counted": 1,

```

Comment:
```suggestion
                    "max_counted": 2,
```

Reply:
Because there are 3 prerequisites, this makes more sense.
This will make it show the number out of 3 instead of out of 2.